styles/styles.css

```
@@ -0,0 +1,288 @@
+@font-face {
+  font-family: "PT Sans";
+  font-style: normal;
+  font-weight: 400;
+  src: url("fonts/pt-sans-regular.woff2") format("woff2");
```

Неверно написан относительный путь к файлу шрифта (нужно исправить во всех `@font-face`) и браузер шрифт не подключит. Должно быть так:
```suggestion
  src: url("../fonts/pt-sans-regular.woff2") format("woff2");
```
[Абсолютные и относительные пути](https://up.htmlacademy.ru/htmlcss/34/book/2.3)